### ISSUE TITLE
Prism shots do not ricochet

### ISSUE THREAD

**Issue by ghost** [phrohdoh/OpenRA#30]
In RA2, a prism tower/tank shot could ricochet to nearby units, for massive damage and destruction!

Revision: 41d9a20

**Comment by penev92** [comment]
Also to the ground.

**Comment by GraionDilach** [comment]
I want to code this, already have an idea to merge this with the Firestorm Reaper missile splitting logic.

**Comment by GraionDilach** [comment]
Yes, I'll be doing this, looks simpler than expected, although I'll go this way:
- first implement a way to fire a weapon as a superweapon for TS EMP Cannon
- then implement TS Cluster Missile as ^
- then add Firestorm Reaper missile splits with this along.

Step 2 is done but untested, doing step 1 atm.

**Comment by Mailaender** [comment]
Reverse duplicate of https://github.com/OpenRA/ra2/issues/133.
